pkg/capabilities/consensus/ocr3/aggregators/reduce_aggregator.go

```diff
@@ -0,0 +1,504 @@
+package aggregators
+
+import (
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"math"
+	"math/big"
+	"sort"
+	"strconv"
+	"time"
+
+	"github.com/shopspring/decimal"
+	"google.golang.org/protobuf/proto"
+
+	ocrcommon "github.com/smartcontractkit/libocr/commontypes"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	"github.com/smartcontractkit/chainlink-common/pkg/values/pb"
+)
+
+const (
+	AGGREGATION_METHOD_MEDIAN = "median"
```

Comment:
Maybe add a comment that it is not a true median (no averaging of two middle values)

Reply:
Good call out. Added words to AggregationField struct.